tests/errors/semantic/non_blocking/ARRAYS_IN_MULTIPLE_RETURNS.py

```diff
@@ -0,0 +1,7 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring, no-value-for-parameter
```

Comment:
Why do you disable `no-value-for-parameter`? Please don't disable pylint errors without good reason